benchmark/.Project.toml

```diff
@@ -0,0 +1,8 @@
+[deps]
```

Comment:
Why do we have two Project.toml here? (`.Project.toml` and `Project.toml`)

Reply:
Thanks for catching this; Project.toml should be removed.

We separately have `.Project.toml` because `Pkg.add("MadNLP")` in the script adds MadNLP entry to the `Project.toml` file. So, locally running the benchmark and then pushing it to repo can cause an unwanted change in the Project.toml file.

Reply:
I see, it makes sense indeed!